kong/templates/nginx_kong.lua

```diff
@@ -102,6 +102,9 @@ server {
     ssl_session_timeout 10m;
     ssl_prefer_server_ciphers on;
     ssl_ciphers ${{SSL_CIPHERS}};
+> if ssl_dhparam then
+    ssl_dhparam ${{SSL_DHPARAM}};
```

Comment:
Here, I simply want to link to the comment I wrote at https://github.com/Kong/kong/pull/4687#discussion_r290872846 which applies just as much to this change.